pom.xml

```diff
@@ -26,9 +26,9 @@
         <module>bom-2.289.x</module>
         <module>sample-plugin</module>
     </modules>
-    <scm>
-        <connection>scm:git:git@github.com/${gitHubRepo}.git</connection>
-        <developerConnection>scm:git:ssh://git@github.com/${gitHubRepo}.git</developerConnection>
+    <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
+        <connection>scm:git:https://github.com/${gitHubRepo}.git</connection>
+        <developerConnection>scm:git:git@github.com:${gitHubRepo}.git</developerConnection>
```

Comment:
FYI https://github.com/jenkinsci/archetypes/pull/267 though I doubt it matters either way.